tests/python/PyUnusedCodeBearTest.py

```diff
@@ -52,3 +65,15 @@
                 invalid_files=[invalid_imports,
                                invalid_from_import],
                 settings={'remove_all_unused_imports': False})
+
+PyUnusedVariableTest = verify_local_bear(
+                PyUnusedCodeBear,
+                valid_files=[without_useless_variables],
+                invalid_files=[with_useless_variables],
+                settings={'remove_unused_variables': True})
+
+PyUnusedVariableTest = verify_local_bear(
```

Comment:
I think we should differentiate the variable name here, like `PyIgnoreVariableTest` or something.